src/components/Component/Component.tsx

```diff
@@ -136,6 +139,7 @@ export const Component: React.FC<ComponentProps> = ({
         <div className={b()}>
             <div className={b('header')}>
                 <h1 className={b('title')}>{component.title}</h1>
+                {Boolean(maintainers.length) && <HeaderMaintainerList maintainers={maintainers} />}
```

Comment:
<img width="617" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/26fdb5a6-cb1f-41cd-a4cf-4f3a712b5331">
<img width="521" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/c71c8eb4-1143-47f8-8296-f3622aa6197a">

Reply:
i fixed maintainers block for `sm` screen. i couldn't find the designs for `md`, so i just hide user name but still have some issues with `DropdownMenu` title because it's a bit longer than others.

<img width="439" alt="Screenshot 2024-06-08 at 18 20 57" src="https://github.com/gravity-ui/landing/assets/579459/4d1f55cf-c760-4e15-a12c-df5599e9e56d">
<img width="640" alt="Screenshot 2024-06-08 at 18 20 39" src="https://github.com/gravity-ui/landing/assets/579459/86c91dda-da92-4101-8a2f-c27f39649c9f">

Reply:
I think we need to redesign this block, if a component has many maintainers it looks broken. It might be better to shorten the list of maintainers and show "+ N maintainers".

@vvtimofeev what do you think about it?

<img width="734" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/cdbf0e5e-69cb-489f-8b62-1e26fb8a5118">

Reply:
We designed this block with the proposal that each component should have a maximum of 3 maintainers (because they are maintainers, not contributors), but in fact, usually there are only 1 or 2. Do we really have more than two maintainers in any component?

Reply:
@vvtimofeev okay, but anyway we need to trim more than 3 maintainers and show a text like `+N` with link to full list.

And if the component name would be longer, then even 1 or 2 maintainers this block will look bad.

Reply:
As of now, there's no more than one maintainer for each component of uikit. Let's merge it and think about possibility of more than one maintainer later.

Reply:
> @vvtimofeev okay, but anyway we need to trim more than 3 maintainers and show a text like `+N` with link to full list.
> 
> And if the component name would be longer, then even 1 or 2 maintainers this block will look bad.

Is it OK with you, if we just add logic to ignore any maintainers after 3?

Reply:
> Is it OK with you, if we just add logic to ignore any maintainers after 3?

yeap

Reply:
@lxndr If you don't mind, please add a logic constraint to limit the number of maintainer logos to 3 images.

Reply:
done